llvm-external-projects/iree-dialects/test/Dialect/iree_linalg_ext/tile_and_decompose_winograd_input.mlir

```diff
@@ -0,0 +1,106 @@
+// RUN: iree-dialects-opt --iree-linalg-ext-tile-and-distribute-winograd-input %s | FileCheck %s
+
+#map = affine_map<(d0)[s0, s1] -> (1, -d0 + s1)>
+#map1 = affine_map<(d0)[s0, s1] -> (32, -d0 + s1)>
+func.func @winograd_input_transform(%arg0: tensor<1x10x10x1280xf32>) -> tensor<8x8x1x2x2x1280xf32> {
+  %c0 = arith.constant 0 : index
+  %c1 = arith.constant 1 : index
+  %c1280 = arith.constant 1280 : index
+  %c32 = arith.constant 32 : index
+  %0 = tensor.empty() : tensor<8x8x1x2x2x1280xf32>
+  %1 = scf.for %arg1 = %c0 to %c1 step %c1 iter_args(%arg2 = %0) -> (tensor<8x8x1x2x2x1280xf32>) {
+    %2 = affine.min #map(%arg1)[%c1, %c1]
+    %3 = scf.for %arg3 = %c0 to %c1280 step %c32 iter_args(%arg4 = %arg2) -> (tensor<8x8x1x2x2x1280xf32>) {
+      %4 = affine.min #map1(%arg3)[%c32, %c1280]
+      %extracted_slice = tensor.extract_slice %arg0[%arg1, 0, 0, %arg3] [%2, 10, 10, %4] [1, 1, 1, 1] : tensor<1x10x10x1280xf32> to tensor<?x10x10x?xf32>
+      %extracted_slice_0 = tensor.extract_slice %0[0, 0, %arg1, 0, 0, %arg3] [8, 8, %2, 2, 2, %4] [1, 1, 1, 1, 1, 1] : tensor<8x8x1x2x2x1280xf32> to tensor<8x8x?x2x2x?xf32>
+      %5 = iree_linalg_ext.winograd.input_transform output_tile_size(6) kernel_size(3) tensor_format("nhwc") ins(%extracted_slice : tensor<?x10x10x?xf32>) outs(%extracted_slice_0 : tensor<8x8x?x2x2x?xf32>) -> tensor<8x8x?x2x2x?xf32>
+      %inserted_slice = tensor.insert_slice %5 into %arg4[0, 0, %arg1, 0, 0, %arg3] [8, 8, %2, 2, 2, %4] [1, 1, 1, 1, 1, 1] : tensor<8x8x?x2x2x?xf32> into tensor<8x8x1x2x2x1280xf32>
+      scf.yield %inserted_slice : tensor<8x8x1x2x2x1280xf32>
+    }
+    scf.yield %3 : tensor<8x8x1x2x2x1280xf32>
+  }
+  return %1 : tensor<8x8x1x2x2x1280xf32>
+}
+
+// CHECK-DAG:    #map = affine_map<(d0)[s0, s1] -> (1, -d0 + s1)>
+// CHECK-DAG:    #map1 = affine_map<(d0)[s0, s1] -> (32, -d0 + s1)>
+// CHECK-DAG:    #map2 = affine_map<(d0) -> (d0 * 6)>
+// CHECK-DAG:    #map3 = affine_map<(d0) -> (-d0 + 10, 8)>
+// CHECK:        module {
```

Comment:
Don't need to check the wrapping module here -- it allows us to avoid one level of indent.

Reply:
Thanks will remove those checks